zone/client.cpp

```diff
@@ -6254,6 +6255,53 @@ void Client::DragCorpses()
 	}
 }
 
+void Client::ConsentCorpses(const char* consent_name, bool deny)
+{
```

Comment:
You wanna add a null check? I know none of the current code should cause issues, but it's not like this is in a hot path or anything, so whatever, better safe than sorry :P